tests/jballerina-unit-test/src/test/resources/test-src/types/stream/stream-value.bal

```diff
@@ -451,3 +451,55 @@ function testInvalidCast() {
     stream<Foo, error?> fooStream = fooArr.toStream();
     stream<Foo, error> barStream = <stream<Foo, error>>fooStream;
 }
+
+function testBasicStreamType() {
+    boolean testPassed = true;
+    stream<int> emptyStream1 = new;
+    stream<int> emptyStream2 = new stream<int>();
+    stream<int, ()> emptyStream3 = new;
+    stream<int, error?> emptyStream4 = new;
+    stream<int, ()> emptyStream5 = new stream<int, ()>();
+    stream<int, error?> emptyStream6 = new stream<int, error?>();
+    var emptyStream7 = new stream<int>();
+    var emptyStream8 = new stream<int, ()>();
+    var emptyStream9 = new stream<int, error?>();
+
+    stream a1 = emptyStream1;
+    stream a2 = emptyStream2;
+    stream a3 = emptyStream3;
+    stream a4 = emptyStream4;
+    stream a5 = emptyStream5;
+    stream a6 = emptyStream6;
+    stream a7 = emptyStream7;
+    stream a8 = emptyStream8;
+    stream a9 = emptyStream9;
```

Comment:
Let's add some tests with a stream of errors and stream of unions

Reply:
Added by 0ebbae52a7f95f1bacad73c6d05ae6d955efa5e6